scripts/generate-dotnet.ps1

```diff
@@ -19,4 +19,9 @@ $LicenseHeader = Join-Path $RootDir src cs LicenseHeader.txt
 
 ClangSharpPInvokeGenerator -f $MsQuicHeader -n Microsoft.Quic -o $MsQuicGeneratedSource -m MsQuic -l msquic `
     -c exclude-enum-operators -r _SOCKADDR_INET=QuicAddr -c generate-macro-bindings -h $LicenseHeader `
-    -e QUIC_UINT62_MAX
+    -e QUIC_UINT62_MAX -e MsQuicOpen2 -e QUIC_API_VERSION_1 -D QUIC_API_ENABLE_INSECURE_FEATURES
```

Comment:
What does the `-e` mean?

Reply:
Exclude. Makes it so those definitions are not included.